test/Libraries/Microsoft.Extensions.AI.AotCompatibility.TestApp/Program.cs

```diff
@@ -0,0 +1,24 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#pragma warning disable CA1031 // Do not catch general exception types
+#pragma warning disable S108 // Nested blocks of code should not be left empty
+#pragma warning disable S125 // Remove this commented out code
+
+using Microsoft.Extensions.AI;
+
+// Use types from each library
```

Comment:
You don't actually need to use types from each library for this project to catch problems. `TrimmerRootAssembly` will ensure the whole library is analyzed.

Reply:
Without this I was getting warnings about having references to assemblies that weren't being used.

Reply:
😮. Can you list the warnings here? That seems surprising - I'd like to know where they came from.

Reply:
```
    CSC : error RT0002: ProjectReference ..\..\..\src\Libraries\Microsoft.Extensions.AI.AzureAIInference\Microsoft.Extensions.AI.AzureAIInference.csproj can be removed
    CSC : error RT0002: ProjectReference ..\..\..\src\Libraries\Microsoft.Extensions.AI.Ollama\Microsoft.Extensions.AI.Ollama.csproj can be removed
    CSC : error RT0002: ProjectReference ..\..\..\src\Libraries\Microsoft.Extensions.AI.OpenAI\Microsoft.Extensions.AI.OpenAI.csproj can be removed
```

Presumably it's this:
https://github.com/dotnet/extensions/blob/7fe79a39d7e28f46eaa51d82a3d433bff73bd3cd/eng/MSBuild/Analyzers.props#L25

Like I said, lots of dubious analyzers.